oauth2client/contrib/_appengine_ndb.py

```diff
@@ -76,9 +76,9 @@ def _validate(self, value):
         """
         _LOGGER.info('validate: Got type %s', type(value))
         if value is not None and not isinstance(value, client.Flow):
-            raise TypeError('Property %s must be convertible to a flow '
-                            'instance; received: %s.' % (self._name,
-                                                         value))
+            raise TypeError('Property {0} must be convertible to a flow '
```

Comment:
prefer breaking after the opening `(`, so this doesn't get so scrunched up at the end of the line.